src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/dnn-persona-bar-page-treeview/src/_PersonaBarTreeInContextMenu.jsx

```diff
@@ -57,37 +50,81 @@ export class PersonaBarTreeInContextMenu extends Component {
                 break;
         }
     }
-    sort(items, column, order) {
```

Comment:
Having removed the `sort` method, are you sure it isn't called anywhere that this component is used?

Reply:
yes, used only once